pallet/staking/src/lib.rs

```diff
@@ -181,18 +192,18 @@ pub mod pallet {
 	#[pallet::getter(fn collator_of)]
 	pub type Collators<T: Config> = StorageMap<_, Twox64Concat, T::AccountId, Perbill>;
 
-	/// Current stakers' exposure.
+	/// Previous stakers' exposure.
 	#[pallet::storage]
 	#[pallet::unbounded]
-	#[pallet::getter(fn exposure_of)]
-	pub type Exposures<T: Config> =
+	#[pallet::getter(fn previous_exposure_of)]
+	pub type PreviousExposures<T: Config> =
 		StorageMap<_, Twox64Concat, T::AccountId, Exposure<T::AccountId>>;
 
-	/// Next stakers' exposure.
+	/// Active stakers' exposure.
 	#[pallet::storage]
 	#[pallet::unbounded]
-	#[pallet::getter(fn next_exposure_of)]
-	pub type NextExposures<T: Config> =
```

Comment:
NextExposures is for elect and store info related to next session, should be be deleted IMO